scalameter-core/src/main/scala/org/scalameter/InvocationCount.scala

```diff
@@ -3,29 +3,24 @@ package org.scalameter
 import java.io.File
 import org.scalameter.Key._
 import org.scalameter.execution.invocation.InvocationCountMatcher
-import org.scalameter.execution.invocation.instrumentation.MethodInvocationCounter
-import org.scalameter.execution.invocation.instrumentation.MethodSignature
-import org.scalameter.execution.invocation.instrumentation.Instrumentation
-import org.scalameter.utils.ClassPath
-import scala.collection.Seq
-import scala.collection.mutable
+import org.scalameter.execution.invocation.instrumentation.{Instrumentation, MethodInvocationCounter, MethodSignature}
+import scala.collection.{Seq, mutable}
 
 
 /** Mixin for all [[org.scalameter.Measurer]] implementations that perform any kind of
  *  method invocation counting.
  */
-trait InvocationCount extends Measurer {
+trait InvocationCount extends Measurer[Double] {
```

Comment:
I expected that the measurer would produce `Map[String, Int]` here?
Then:
- clients can use the `map` operation to extract desired information from this map, such as the total count across all methods
- you could also provide factory methods on the `InvocationCount` companion object to return the total method invocation count

I guess your plan is to do this refactoring in the next PR?